life_hc/MySQL/Gangs/fn_updateGang.sqf

```diff
@@ -30,7 +31,33 @@ switch (_mode) do {
     };
 
     case 1: {
-        _query = format ["UPDATE gangs SET bank='%1' WHERE id='%2'",([(_group getVariable ["gang_bank",0])] call HC_fnc_numberSafe),_groupID];
+        params [
+            "",
+            "",
+            ["_deposit",false,[false]],
+            ["_value",0,[0]],
+            ["_unit",objNull,[objNull]],
+            ["_cash",0,[0]]
+        ];
+
+        private _funds = _group getVariable ["gang_bank",0];
+        if (_deposit) then {
+            _funds = _funds + _value;
+            _group setVariable ["gang_bank",_funds,true];
+            [1,format["Successfully deposited $%1.",_value]] remoteExecCall ["life_fnc_broadcast",remoteExecutedOwner];
+            _cash = _cash - _value;
+        } else {
+            if (_value > _funds) exitWith {
+                [1,"There are not enough funds in the bank for this withdrawal."] remoteExecCall ["life_fnc_broadcast",remoteExecutedOwner];
+                breakTo "main";
```

Comment:
Would be more elegant to simply `breakOut ""`, and remove the scopeName above, this usage of breakTo is pretty spaghetti